ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -810,6 +812,65 @@ setNameCache :: IORef NameCache -> HscEnv -> HscEnv
 #endif
 setNameCache nc hsc = hsc { hsc_NC = nc }
 
+#if MIN_VERSION_ghc(9,3,0)
+-- This function checks then important property that if both p and q are home units
```

Comment:
Just for my understanding: what are the expected GHC versions to which this fix will be backported?

```suggestion
-- This function checks the important property that if both p and q are home units
```